config/longrun_configs/slabplanet_aqua_coupler_sf_evolve_ocn.yml

```diff
@@ -9,6 +9,7 @@ h_elem: 6
 land_albedo_type: "function"
 mode_name: "slabplanet_aqua"
 moist: "equil"
+ode_algo: "ARS343"
```

Comment:
I thought we only want to use ARS343 for runs with diagnostic EDMF. Do we want it for this run too?

Reply:
We're looking into possible issues when there is precipitation, with the `SSPKnoth` scheme, so I have reset this `ode_algo` to `ARS343` in the meantime.